db/migrate/20200309055502_drop_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropTable < ActiveRecord::Migration[6.0]
+  def change
+    drop_table :treatments
```

Comment:
Same thing here - the treatments table doesn't exist so an error is thrown. You can remove these two migrations to get yours to work.